lips/lip-1.mdx

```diff
@@ -11,149 +11,122 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
-**reference_id**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object.  The reference ID is used as a unique identifier of the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be unique, and formatted as “<creator_vasp_onchain_address_bech32>_<unique_id>”. For example, ”lbr1x23456abcd_seqABCD“. As an example, see the reference_id field on the [PaymentObject](#paymentobject). Unicode utf-8 encoded. "unique_id" should be 64 or fewer characters.
+The remainder of this document details the specifications for Commands, Objects and the typical and exceptional flows to define Payments and their Commands.
 
-# Overall Off-Chain Properties
-The Off-Chain Protocol formulates methodologies to provide the following high-level properties:
+---
+# Specification
+---
 
-### Channel Establishment
-The Off-Chain Protocol allows two entities A and B to form a communication Channel for manipulating a consistent database of Objects and agreeing on Commands that manipulate those Objects.
+### Terminology
 
-### Command Issuance By Either Party
-Both ends of the Channel can issue Commands upon Shared Objects.
+**Object**: A record. A PaymentObject is an example of an Object representing a payment.
 
-### Object Versioning
-To ensure a consistent view of Object states, every Object is versioned. Mutations may only occur upon the latest Version of an Object.
+**Command**: An instruction sent over a Channel to mutate/create one or more Objects. In the case of a mutation, this Command will depend upon the current value of one of more existing Shared Objects.
 
-### Concurrency Control
-The Off-Chain Protocol manages concurrent requests on Objects and ensures that Object states will not become out of sync.
+**Channel**: The communication path between a pair of entities who execute Commands and track the evolution of a set of Shared Objects.
 
-### Eventual Consistency
-The Off-Chain Protocol ensures eventual consistency of Shared Objects.
+**Shared Object**: All Objects contained within a Command are logically shared, meaning that each VASP has a copy of the Object and may create a new Command to modify it. For example, during the lifecycle of a `KycDataObject`, both VASPs will add information to it. The protocol does not support simultaneous updates to Objects. Rather, VASPs must take turns updating the Object.
 
-# Basic Off-Chain Building Blocks
+**Reference ID**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be global unique. We recommend use a 128 bits long UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122).
```

Comment:
```suggestion
**Reference ID**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be globally unique. It is recommended to use a 128 bit long UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122).
```

Reply:
Let's be explicit about whether "-" is needed here.